server/journeys/screen/check-answers/tests.test.ts

```diff
@@ -0,0 +1,86 @@
+import { Locals, Request } from 'express'
+import { v4 as uuidv4 } from 'uuid'
+import { agent as request } from 'supertest'
+import { getByRole } from '@testing-library/dom'
+import { appWithAllRoutes } from '../../../routes/testutils/appSetup'
+import testRequestCaptor from '../../../routes/testutils/testRequestCaptor'
+import { JourneyData } from '../../../@types/express'
+import { TEST_PRISONER } from '../../../routes/testutils/testConstants'
+import createTestHtmlElement from '../../../routes/testutils/createTestHtmlElement'
+
+const uuid = uuidv4()
+
+const MOCK_INPUT_RADIO = { code: 'AAA', description: 'Selected radio option' }
+const MOCK_INPUT_TEXT_MULTI = `Text
+
+    • Bullet 1
+    • Bullet 2
+    • Bullet 3
+    
+    Paragraph
+    
+    <script>alert('concerns');</script>
+    
+    <button>this button should be escaped</button>`
+
+const MOCK_INPUT_TEXT_MULTI_EXPECTED = `Text<br>\n      <br>\n          • Bullet 1<br>\n          • Bullet 2<br>\n          • Bullet 3<br>\n          <br>\n          Paragraph<br>\n          <br>\n          &lt;script&gt;alert('concerns');&lt;/script&gt;<br>\n          <br>\n          &lt;button&gt;this button should be escaped&lt;/button&gt;`
```

Comment:
if these test values are re-usable for other tests, we might move them to the test constants file?

Reply:
I was hoping to do this, glad you think it'll useful too!